winml/lib/Api.Core/FeatureDescriptorFactory.cpp

```diff
@@ -473,14 +473,12 @@ GetTensorType(
        has_unsupported_image_metadata);
 
   if (is_tensor_improperly_annotated_as_image) {
-#ifdef LAYERING_DONE
-    TraceLoggingWrite(winml_trace_logging_provider,
+    TraceLoggingWrite(_winmla::winml_trace_logging_provider,
```

Comment:
Where is `_winmla` defined in the source code? I couldn't find in the layer_dev branch, is this generated by `MIDL_INTERFACE`s in WinMLAdapter.h?

Reply:
_winmla is defined in winml/lib/Common/inc which is shorthand for ::Windows::AI::MachineLearning::Adapter namespace

Reply:
Ok, found it 😄 

You should rename it, there's a C++ rule that names starting with an underscore shouldn't be placed in the global namespace:

```
Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.
```

(From https://stackoverflow.com/a/228797)

Reply:
@walrusmcd , what do you think ?